package.json

```diff
@@ -66,7 +66,7 @@
     "read": "^1.0.7",
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
-    "redux-firestore": "^0.14.0",
+    "redux-firestore": "https://registry.npmjs.org/redux-firestore-forked/-/redux-firestore-forked-0.14.0.tgz",
```

Comment:
As per Slack ([ref](https://doesitsparkle.slack.com/archives/C01JHLMSFND/p1611102455057800?thread_ts=1611083128.047000&cid=C01JHLMSFND)):

> we definitely don’t want to be loading the forked dependency from a published npm dependency in the way you have done it there.
>
> npm / package.json support loading dependencies directly from a github repository. See https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a
>
> The ideal workflow here would be that the forked repository would be under the sparkletown organisation, and then we can add that repo/the branch with the fix to our dependencies.

Reply:
Valid point and suggestion

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1047/commits/eb63c06804f586af23cb737893c27b4da97e34b7